src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -239,6 +239,13 @@ internal MetricPointStatus MetricPointStatus
             return this.histogramBuckets;
         }
 
+        internal MetricPoint Copy()
```

Comment:
I think we need `readonly` on the method definition. Otherwise caller might incur a defensive copy.

Reply:
done.